client/extensions/woocommerce/app/dashboard/pre-setup-view.js

```diff
@@ -10,46 +10,100 @@ import React, { Component, PropTypes } from 'react';
  * Internal dependencies
  */
 import AddressView from 'woocommerce/components/address-view';
+import {
+	areSettingsGeneralLoading,
+	getStoreLocation,
+} from 'woocommerce/state/sites/settings/general/selectors';
+import { errorNotice } from 'state/notices/actions';
+import { getSelectedSiteWithFallback } from 'woocommerce/state/sites/selectors';
```

Comment:
Thank you :) Fixed in https://github.com/Automattic/wp-calypso/pull/15463/commits/bc466ca37e62386bab22f2d209ed4199fce059dd